### PR TITLE
Remove misleading visible:false option for waitAny()

### LINKED ISSUE #4722: Disallow `visible: false` in waitAny()
Current `waitAny()` parameters are misleading.
`visible: false` should be treated as "element must not be visible".
However, as we don't actually use this in tests, let's remove this option, and only leave `true | undefined`.
Note: `waitAll()` treats all 3 options (true, false and undefined) correctly.

_Originally posted by @rrrooommmaaa in https://github.com/FlowCrypt/flowcrypt-browser/pull/4708#discussion_r979285764_

### PR DESCRIPTION
**Pull request by rrrooommmaaa**
This PR removes unsupported `visible: false` option in `waitAny()` (all the tests use `visible: true` or `visible: undefined`, or `waitAll()` instead

close #4722

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: test/source/browser/controllable.ts [hunk 1]
```diff
@@ -76,8 +76,9 @@ abstract class ControllableBase {
     this.log(`wait_all:7:${selectors.join(',')}`);
   };
 
-  public waitAny = async (selector: string | string[], { timeout = TIMEOUT_ELEMENT_APPEAR, visible = true }: { timeout?: number, visible?: boolean } = {}): Promise<ElementHandle> => {
-    return await this.waitAnyInternal(this.selsAsProcessedArr(selector), { timeout, visible });
+  public waitAny = async (selector: string | string[], properties: { timeout?: number, visible: true | undefined } | { timeout?: number } = {}): Promise<ElementHandle> => {
```
Comment (ioanmo226) [inline test/source/browser/controllable.ts:79]: Why `visible` type becomes `true | undefined`? Any problem with `visible?: boolean`?

Comment (ioanmo226) [reply test/source/browser/controllable.ts:79]: Never mind, Just figured out because visible `false` option should not be passed.

**Review by ioanmo226 — APPROVED** [review]
👍

File: test/source/browser/controllable.ts (hunk 1 above)
Comment (rrrooommmaaa) [reply test/source/browser/controllable.ts:79]: In `waitAll` the meanings are:
`true` element must exist and be visible
`false` element must exist and be not visible.
`undefined` element must exist

`waitAny` doesn't support the option: element must exist and be not visible.
This PR makes `waitAny` in line with `waitAll` meanings

**Pull request merged**